src/applications/static-pages/coronavirus-chatbot-utils.js

```diff
@@ -0,0 +1,21 @@
+function disableButtons(event) {
```

Comment:
I'm not sure the history behind this, but we might be able to find a more elegant solution. Until we get to the elegant solutioning phase, though, :see_no_evil:

Reply:
Agreed!